package.json

```diff
@@ -51,6 +51,7 @@
     "js-beautify": "^1.7.5",
     "jstransformer-marked": "^1.0.3",
     "jstransformer-nunjucks": "^0.5.0",
+    "marked": "^0.5.1",
```

Comment:
marked is already installed with `jstransformer-marked` so no need to install it again i don't think

Reply:
I think if this is the first time we're requiring marked within our codebase we should list it as an explicit dependency.

Reply:
we already require it in the extract-headings, but that's fine, we can list it as a dependency